### PR TITLE
Implement Basic Assets Page

### PR DESCRIPTION
**Pull request by Vasyl3611**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -17,3 +16,33 @@ body {
...
+    background: linear-gradient(270deg, #ffb6b1 1.89%, #ff7067 100%);
+    transform: translateY(-50%);
+}
+
+.description {
+    margin: 10px 0 35px;
+    font-weight: 250;
+    font-size: 1rem;
+    line-height: 1.1875rem;
+    color: #444444;
```
Comment (PrytulaAlexandr) [inline src/App.css:47]: Better use var(--font-color-six)

File: src/App.css [hunk 2]
```diff
@@ -17,3 +16,33 @@ body {
     max-width: 1920px;
     margin: 0 auto;
 }
+
+.titleWithBorder {
+    position: relative;
+    font-weight: 700;
+    font-size: 2.18rem;
+    line-height: 42px;
+    color: #262626;
```
Comment (PrytulaAlexandr) [inline src/App.css:25]: Better use var(--font-color-primary)

**Review by PrytulaAlexandr — APPROVED** [review]
Nice work👍 But I have some minor comments for you, look, please.

File: src/App.css (hunk 1 above)
Comment (Vasyl3611) [reply src/App.css]: Done

File: src/App.css (hunk 2 above)
Comment (Vasyl3611) [reply src/App.css]: Done

**Pull request merged**